src/features/groups/components/CreateGroupButton/CreateGroupDrawerContent.tsx

```diff
@@ -8,34 +8,45 @@ import {
   DrawerTitle,
   Input,
 } from '@/components/ui';
+import { createGroup } from '@/lib/api/group/client';
 
-import { FC } from 'react';
+import { FormEvent, useState } from 'react';
 
-interface ICreateGroupDrawerContentProps {
-  handleCreateGroup: () => void;
-}
+export const CreateGroupDrawerContent = () => {
+  const [groupName, setGroupName] = useState('');
```

Comment:
Could you add `<IGroup['name']>`, please?
Currently, the type is inferred as a simple `string`, not referring to the type of group name.